code/lib/cli/src/babel-config.ts

```diff
@@ -52,48 +101,29 @@ export const generateStorybookBabelConfig = async ({ target }: { target: string
     },
   ]);
 
-  const added = ['@babel/preset-env'];
-  const presets: (string | [string, any])[] = [['@babel/preset-env', { targets: { chrome: 100 } }]];
-
-  if (typescript) {
-    added.push('@babel/preset-typescript');
-    presets.push('@babel/preset-typescript');
-  }
-
-  if (jsx) {
-    added.push('@babel/preset-react');
-    presets.push('@babel/preset-react');
-  }
-
-  const contents = JSON.stringify(
-    {
-      sourceType: 'unambiguous',
-      presets,
-      plugins: [],
-    },
-    null,
-    2
-  );
+  const dependencies = getBabelPresets({ typescript, jsx });
 
   logger.info(`Writing file to ${location}`);
-  await writeFile(location, contents);
+  await writeBabelConfigFile({ location, typescript, jsx });
 
   const { runInstall } = await prompts({
     type: 'confirm',
     initial: true,
     name: 'runInstall',
-    message: `Shall we install the required dependencies now? (${added.join(', ')})`,
+    message: `Shall we install the required dependencies now? (${dependencies.join(', ')})`,
   });
 
   if (runInstall) {
```

Comment:
`runInstall` not only installs the dependencies but adds them to `package.json`. That means that if you don't install them, you'll be in an inconsistent state where the config file has been generated but even running `npm install` won't get you the deps you need. This is already warned about below, but it's a weird behavior and I think even if the user opts out of installing the deps they should still get added to `package.json.

Reply:
I didn't change the behavior of this code, but I could try and make this change if you like!

Some context:
- This code is NOT executed as part of `storybook init` – the babel generation is handled differently and the dependencies are installed correctly
- This code is executed as part of `automigrate`
- This code is executed when users run `storybook babelrc`

Normally we only add dependencies to the package.json without installing when we know exactly which version to put there (e.g. from versions.ts), but in this case, if we were to skip installing, given that we have no idea which version to put, we'd need to either fetch the latest version using their package manager or add `"latest"` to package.json.

Optionally, we could just run install without prompting the user

Reply:
Yup, I saw that. Do as you see fit!

Reply:
Removed the prompt! Thanks for catching this! I think it's better now